contracts/MarginCalculator.sol

```diff
@@ -488,6 +488,21 @@ contract MarginCalculator is Ownable {
         return (true, debtPrice, dust[vaultDetails.shortCollateralAsset]);
     }
 
+    /**
+     * @notice an external callable _getMarginRequired()
```

Comment:
Should we use a shorter version of the `_getMarginRequired` description here? instead of saying it's calling another function (kind of meaningless).

Reply:
Agree 🙂

Reply:
What other comment u recommend ?

Reply:
I suggest the following: 
`getMarginRequired`: calculate the amount of collateral needed for a vault
`_getMarginRequired`:  calculate the amount of collateral needed for a vault, taking vaultDetail for gas saving